### ISSUE TITLE
[Squid] Review txn fees for sudo account

### ISSUE THREAD

**Issue by saboonikhil** [zeitgeistpm/zeitgeist#482]
[Older data] There seem to be no txn fees incurred against `dDyymrYQMxP5ksYCLXQjv9oAYiQdJFE9Ampgv8cYxc4qXpTNH` for executing following extrinsics via sudo call:
- `balances.setBalance` at [#14739](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/14739) 
- `predictionMarkets.adminDestroyMarket` at [#15536](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/15536)/[#15544](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/15544)/[#15762](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/15762)/[#15784](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/15784)/[#15838](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/15838)

However, txn fees has been considered for `balances.transferKeepAlive` (non-sudo call) at [#13829](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/explorer/query/13829)

Is it safe to avoid calculating txn fees for sudo calls?

**Comment by sea212** [comment]
Yes. As stated by @maltekliemann in the meeting, the superuser account does not pay any fees.
